core/server/utils/startup-check.js

```diff
@@ -5,6 +5,7 @@ var packages = require('../../../package.json'),
     mode = process.env.NODE_ENV === undefined ? 'development' : process.env.NODE_ENV,
     appRoot = path.resolve(__dirname, '../../../'),
     configFilePath = process.env.GHOST_CONFIG || path.join(appRoot, 'config.js'),
+    config = require('../config'),
```

Comment:
Hm, that means I'd also have to duplicate the config option default value merge here as well. That was the reason I included it here.